README.md

```diff
@@ -1,80 +1,190 @@
+
+*Tip: Check out the documentation for the accounts packages. Meteor Accounts also has a [project page](https://www.meteor.com/accounts) on meteor.com*
+
+*Tip: Put your *loginButtons* in the actionBar.html file to get our custom style for the sign in link*
+
+Now you can sign in, but when you do, the sign in link does not dissapear which is kind of strange. *Accounts-ui* comes with a handy global template helper function, `{{currentUser}}` which holds the object representation of the current user. It wouldn't have been all that hard to write the helper yourself as it simply returns the result of `Meteor.user()`, but it's still kind of neat. With the *currentUser* helper, you should be able to hide the sign in link when a user is signed in.
 
-A meteor application is usually runs by typing `meteor` in the terminal at the root of the application.
-Since we are dependent on **settings.json** to load with our application, we have supplied a **dev.sh** to make it easier for you.
-For you Windows users, there is also a **dev.bat**. Our pleasure!
+Display the following markup when users have signed in to get a sign out button:
 
-### 2. Profile
-1. We are going to take a look at meteors package system, [atmosphere](https://atmospherejs.com).
-Head over there and get `accounts-password` and `accounts-ui` for your project.
-It can be helpful to look at the extended documentation on [docs.meteor.com](http://docs.meteor.com/#/full/accounts_api)
+```html
+<span class="glyphicon glyphicon-log-out" aria-hidden="true"></span>
+```
 
-   To get a login-button simple add: {{> loginButtons}} to the desired template (preferably the actionBar, if currentUser is not logged in).
+To get this button to actually do something, you have to create an event handler in *actonBar.js* that logs you out and redirects you to the homepage. Meteor itself can help with the first using the `Meteor.logout` function, while iron:router has a function called *go* that takes in a route name that can help you accomplish the latter.
 
-   Hint: It might be useful to have a look in **accountsConfig.js** after you have acquired the packages.
+#### 4. Profiles
+##### Basic information
+Now that users can sign in, they should be able to do something with their own profiles. As you can see, we have chosen to not hand out names for our users this time. However, this is something you should change. Open *profile.html* and create fields for the users profile. These should include firstname, lastname, adress and phone. Since it looks kind of weird when you first sign in and don't have a name, feel free to use the function we provided in the [chat-application](https://github.com/EirikV/meteor-workshop/tree/chat-solution) to hand out a temporary name as users sign in.
```

Comment:
As you can see, we have chosen to assign names to new users as we did in the chat-app